base/redhat-8/Dockerfile

```diff
@@ -29,7 +29,7 @@ ARG SCLOUD_URL
 ARG BUSYBOX_URL
 ENV SCLOUD_URL=${SCLOUD_URL} \
     BUSYBOX_URL=${BUSYBOX_URL} \
-    PYTHON_VERSION=3.7.10 \
+    PYTHON_VERSION=3.7.16 \
```

Comment:
> [ansible@a0497ec8f1b3 bin]$ ./splunk cmd python -V
> Python 3.7.11
> [ansible@a0497ec8f1b3 bin]$ 

Python version in the Dockerfile must align with the Python version in Splunk Enterprise.

Reply:
working on getting the Python updated to 3.7.16 in the upcoming releases. Will share the estimated version/date soon.

Reply:
Python version is upgraded to 3.7.16 in the latest 9.0.5 release. closing out this MR.

Reply:
So based on this:

> Python version in the Dockerfile must align with the Python version in Splunk Enterprise.

You are going to do a similar PR to this one that you just closed when you publish a new docker-splunk release for 9.0.5?

Reply:
9.0.5 is already released with the Python version upgraded.